### ISSUE TITLE
UI item hovering is inconsistent

### ISSUE THREAD

**Issue by LordSk** [teeworlds/teeworlds#1771]
Hovering tab buttons or specific other buttons applies a "hover effect". However most other UI items (checkbox, listbox, ...) don't have that effect applied. This can trick the user into thinking they can't interact with said items and makes the overall UI feel inconsistent in that regard.

**Comment by Dune-jr** [comment]
I think checkboxes have this hover effect too, but it is only barely noticeable.

**Comment by sepulzera** [comment]
Furthermore, the language and theme lists in the settings menu won't both display the current setting.
Only the last hovered list will. You need to hover the list to see the current selection -> bad.

**Comment by Dune-jr** [comment]
Unlike ui/gui_buttons.png, ui/icons/menu.png does not have a hover item:
**gui_buttons**
![image](https://user-images.githubusercontent.com/355114/51030666-6da22300-159a-11e9-9586-87ae1d74a62e.png)
**icons/menu**
![menu](https://user-images.githubusercontent.com/355114/51030625-55320880-159a-11e9-812b-16e5e4742c44.png)

I emulated one using:
```
	CUIRect c2 = c;
	c2.x += 2.f;
	c2.y += 2.f;
	c2.w -= 3.f;
	c2.h -= 3.f;
	RenderTools()->DrawUIRect(&c2, vec4(1.f, 1.f, 1.f, 0.25f), CUI::CORNER_ALL, 5.0f);
```
in there:
https://github.com/teeworlds/teeworlds/blob/de4ffee5567ce3970970ee9644c68d5eea4da755/src/game/client/components/menus.cpp#L314-L320

This is the result:
![image](https://user-images.githubusercontent.com/355114/51030578-33388600-159a-11e9-9a40-5ba1374c7f26.png)

While doing so, I noticed there is supposed to be a 60% transparency effect on inactive checkboxes
https://github.com/teeworlds/teeworlds/blob/de4ffee5567ce3970970ee9644c68d5eea4da755/src/game/client/components/menus.cpp#L329-L335

but this doesn't do anything.

What is the best course of action?

**Comment by oy** [comment]
Should probably test MouseInside instead. But I think there's no need to have 2 effects in place.
Probably best to go with the hover effect and drop the transparency one.

**Comment by Dune-jr** [comment]
Fixed by #1968 and #1969

**Comment by oy** [comment]
Sliders, edit-boxes and the grid headers in the server browser could have a hover effect too. Dunno if there's more? Would be good to have it for all UI elements that you can interact with.
